core/shared/src/main/scala/zio/ZTrace.scala

```diff
@@ -33,7 +33,9 @@ final case class ZTrace(
     val stackPrint =
       if (stackTrace)
         s"Fiber:$fiberId was supposed to continue to:" ::
-          this.stackTrace.map(loc => s"  a future continuation at " + loc.prettyPrint)
+          this.stackTrace.map(loc =>
+            s"  a future continuation at " + Option(loc).map(_.prettyPrint).getOrElse("unknown")
```

Comment:
This seems like it would fix the error. But I'm not sure this fix addresses the underlying problem.

`stacktrace` is a `List[ZTraceElement]`. It should not contain `null` elements. I wonder how they get there.

Reply:
I agree and have thought the same thing. We might as well reject this particular change - I might be able to spend some more time getting to the bottom of it. I was kind of after a quick fix to keep ZMX happy and working